app/controllers/games_controller.rb

```diff
@@ -14,17 +13,20 @@ def create
 
   private
 
-  def join_game
-    return if @game.state == 'finished'
-    enlist = Players::Enlist.new(game: @game, player: current_player)
-
-    if enlist.game_full?
-      return render FullGame.template_path
-    else # try to enter the game
-      payload = enlist.assign_game_to_player!
-      set_current_player(enlist.player)
+  def game_params
+    params.require(:game).permit(:name)
+  end
 
-      ActionCable.server.broadcast "game_#{current_player.game_id}", payload
+  def game_template
+    case @game.state
+    when 'awaiting_opponent'
```

Comment:
Since games have states, now, I'd be more confortable is those values were constants instead of magic strings.